src/features/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { ReactElement } from 'react';
+import { ReactComponent as Logo } from '../../assets/icons/footer-logo.svg';
+import Politics from './components/Politics/Politics';
+import Contact from './components/Contact/Contact';
+import { scrollToTop } from '../../common/utils';
+
+const Footer = (): ReactElement => {
```

Comment:
https://medium.com/@RobertoSilvaZ/when-should-i-use-jsx-element-or-react-fc-29e61eb1e754

Imo it should be changed to `Footer: FunctionComponent = () =>`